homeassistant/components/switchbot/switch.py

```diff
@@ -73,7 +73,7 @@ def assumed_state(self) -> bool:
         return True
 
     @property
-    def is_on(self) -> bool:
+    def is_on(self):
         """Return true if device is on."""
         return self._state
```

Comment:
```suggestion
    def is_on(self) -> bool:
        """Return true if device is on."""
        return bool(self._state)
```

Reply:
Isn't it better to use `cast(bool, self._state)`?

Reply:
You defined it as being able to be `None`

![image](https://user-images.githubusercontent.com/195327/127862519-6d32029d-2811-42b7-9e51-4f8b4bb4d71d.png)

Thus it needs to be actually converted.